### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by kmohrf**
this fixes the problems described in #189

### CODE REVIEW

**Comment by kmohrf** [comment]
nice thing that github goes down when the builds are running :(

File: test/test-widgets.js [hunk 1]
```diff
@@ -155,8 +155,8 @@ test('multipleCheckbox', function (t) {
         var field = {
             choices: {
                 one: 'Item one',
-                three: 'Item three',
-                two: 'Item two'
+                two: 'Item two',
```
Comment (ljharb) [inline test/test-widgets.js]: these are in alphabetical order intentionally due to the linter.

File: test/test-widgets.js [hunk 2]
```diff
@@ -230,7 +230,11 @@ test('multipleCheckbox', function (t) {
 test('multipleCheckbox multiple selected', function (t) {
     var w = forms.widgets.multipleCheckbox(),
         field = {
-            choices: { one: 'Item one', three: 'Item three', two: 'Item two' },
+            choices: {
+                one: 'Item one',
+                two: 'Item two',
+                three: 'Item three'
+            },
             value: ['two', 'three']
```
Comment (ljharb) [inline test/test-widgets.js]: iow i think swapping the order of this array should be enough to fix this test.

**Comment by ljharb** [comment]
(please do try to rebase and force push to this branch, and be sure not to close this and open a second PR)

File: test/test-widgets.js (hunk 1 above)
Comment (kmohrf) [inline test/test-widgets.js]: the linter didn’t complain. did i miss something?

Comment (ljharb) [inline test/test-widgets.js]: hm, perhaps i disabled the setting to enforce alphabetized keys - if the travis tests pass then it's good as-is

File: test/test-widgets.js (hunk 2 above)
Comment (kmohrf) [inline test/test-widgets.js]: nope. just changed it locally. if i change the choices list back they way it was and switch to value keys (that’s what you meant right?) the inputs and labels still appear in order of the choices object (one, three, two).

Comment (ljharb) [inline test/test-widgets.js]: i meant keeping "one, three, two", and changing `value` to `['three', 'two']`.

**Comment by ljharb** [comment]
Thanks, this is good as-is - I've rebased and cleaned up the commit messages, and will merge ASAP.

**Pull request merged**